### PR TITLE
Do not shutdown logger context during Node close

### PR DESCRIPTION
**Pull request by vrozov**
*Issue #, if available: Avoid changing logger configuration during tests

*Description of changes:

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/test/java/com/amazon/opendistroforelasticsearch/security/test/helper/cluster/ClusterHelper.java [hunk 1]
```diff
@@ -244,8 +242,6 @@ public final void stopCluster() throws Exception {
 
     private static void closeNode(Node node) {
         try {
-            LoggerContext context = (LoggerContext) LogManager.getContext(false);
-            Configurator.shutdown(context);
```
Comment (debjanibnrj) [inline src/test/java/com/amazon/opendistroforelasticsearch/security/test/helper/cluster/ClusterHelper.java:248]: any reason why we are not shutting down this logger context?

Comment (vrozov) [reply src/test/java/com/amazon/opendistroforelasticsearch/security/test/helper/cluster/ClusterHelper.java:248]: The `LoggerContext` is not cluster specific and it was not created during cluster creation. It must not be shutdown when a node is closed and/or when cluster is stopped. Shutting down `LoggerContext` may cause a change in `Logger` levels and missing log messages that may help with troubleshooting unit tests.

Comment (dinusX) [reply src/test/java/com/amazon/opendistroforelasticsearch/security/test/helper/cluster/ClusterHelper.java:248]: This code is used only by tests . Maybe some tests need this for correct functionality or more verbosity ? 

What is the reason for removing this code ? Are there some logs or output missing ? 

Do you have any example of the difference in output "before/after" to understand how this change helps ?

Comment (vrozov) [reply src/test/java/com/amazon/opendistroforelasticsearch/security/test/helper/cluster/ClusterHelper.java:248]: The code only affects the amount of logging and I would not expect tests to depend on disabling or shutting down logging that makes it less verbose. In addition, the order of tests execution is not defined and one unit test may not rely on another unit test to shutdown the logging context.

The easiest way to see the difference is to change `cas` logger level from `error` to `trace` in `log4j2-test.properties` and run `SSLTest` or `OpenSSLTest` (or any other test suite that creates two or more clusters). After the subject code is called, there would be no entries from the `cas` logger logged.

**Pull request merged**